addons/armaos/functions/fnc_terminal_setPrompt.sqf

```diff
@@ -0,0 +1,21 @@
+params ["_computer"];
+
+private _terminal = _computer getVariable "AE3_terminal";
+
+private _terminalBuffer = _terminal get "AE3_terminalBuffer";
+private _terminalCursorPosition = _terminal get "AE3_terminalCursorPosition";
+private _terminalPrompt = _terminal get "AE3_terminalPrompt";
+
+private _lastBufferLineIndex = (count _terminalBuffer) - 1;
+private _lastBufferLine = _terminalBuffer # (_lastBufferLineIndex);
+
+_lastBufferLine = _lastBufferLine + _terminalPrompt;
+
+_terminalBuffer set [_lastBufferLineIndex, _lastBufferLine];
+
+_terminalCursorPosition = (count _lastBufferLine);
+
+_terminal set ["AE3_terminalBuffer", _terminalBuffer];
+_terminal set ["AE3_terminalCursorPosition", _terminalCursorPosition];
+
+_computer setVariable ["AE3_terminal", _terminal];
```

Comment:
Should be unnecessary, because variables referencing hashmaps are only pointers to the actual hashmap object, not the hashmap itself.

Reply:
removed in https://github.com/y0014984/Advanced-Equipment/pull/90/commits/996717f1cedd397770d94ec876aedb11e65687ad